src/net_processing.cpp

```diff
@@ -1511,6 +1515,11 @@ void PeerManagerImpl::SendPings()
 }
 
 void PeerManagerImpl::RelayTransaction(const uint256& txid, const uint256& wtxid)
+{
+    WITH_LOCK(cs_main, _RelayTransaction(txid, wtxid););
```

Comment:
No need for `;);` -- there's already a terminating `;` in the `WITH_LOCK` `#define`. Could just use a plain `LOCK` instead of `WITH_LOCK` too...